src/components/container/HeartEssentialEight/HeartEssentialEight.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Meta, StoryObj } from "@storybook/react";
+import Layout from "../../presentational/Layout";
+import HeartEssentialEight, {
+  HeartEssentialEightProps,
+} from "./HeartEssentialEight";
+
+const meta: Meta<typeof HeartEssentialEight> = {
+  title: "Container/HeartEssentialEight",
+  component: HeartEssentialEight,
+  parameters: {
+    layout: "fullscreen",
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof HeartEssentialEight>;
+
+const render = (args: HeartEssentialEightProps) => {
+  return (
+    <Layout colorScheme="auto">
```

Comment:
Consider setting up the story to allow for swapping between color schemes for easier review of light/dark/auto modes.

Similarly, the preview state can be set up as a toggle on a single story if you'd like.

An example: https://github.com/CareEvolution/MyDataHelpsUI/blob/main/src/components/container/ConnectDexcom/ConnectDexcom.stories.tsx